references/indexes.md

```diff
@@ -120,24 +127,46 @@ curl \
   --header 'Content-Type: application/json' \
   --header "X-Meili-API-Key: $API_KEY" \
   --data '{
+  "name": "Movies",
+  "schema": {
     "id": ["identifier", "indexed", "displayed"],
     "title": ["displayed", "indexed"],
     "overview": ["displayed", "indexed"],
     "release_date": ["displayed"],
     "poster": ["displayed"]
+  }
 }'
 ```
 
 If no schema has been defined : 
-#### Response: `204 No content`
+#### Response: `201 created`
+```json
+{
+  "name": "Movie",
+  "uid": "3d9pkcnb",
+  "schema": null,
+  "createdAt": "2019-11-20T09:40:33.711476Z",
+  "updatedAt": "2019-11-20T09:40:33.711476Z"
+}
+```
 
 If schema has been given : 
 #### Response: `201 created`
```

Comment:
Since an updateId is given in the returned object, should it not be a `202 Accepted` by consistency with the other routes?

Reply:
True, I will change it